packages/syntax-floating-components/src/Tooltip/Tooltip.module.css

```diff
@@ -0,0 +1,9 @@
+.tooltipContent {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  background-color: var(--color-base-gray-900);
```

Comment:
was there designs for it to always be black/white tooltips?

should this be a variable so we could have "light mode" options as well?

Reply:
There weren't any light mode designs for the tooltip

Reply:
it just feels like the background/text color could be the same props as what Box and Typography has, so they can be reused easier